docs/file_formats/fip.md

```diff
@@ -180,7 +204,13 @@ Data acquisition code that generates data in this format is available from the [
 
 ### Relationship to aind-data-schema
 
-procedures.json documents the relevant fiber probe implantation metadata (stereotaxic coordinates) and viral injection metadata (stereotaxic coordinates, materials). session.json documents the intended measurement (e.g. norepinephrine, dopamine, etc) for each channel of each probe. 
+The FIP file format documents ROI-based measurements from patch cords. To fully interpret this data, additional metadata from aind-data-schema files is required:
+
+* **procedures.json**: Documents implanted fiber locations (stereotactic coordinates) and viral injection metadata (stereotactic coordinates, materials, viral constructs)
+
+* **acquisition.json**: Documents which patch cords were connected to which implanted fibers during the experiment. This is contained in the [connections](https://aind-data-schema.readthedocs.io/en/latest/components/connections.html#connection) list in the [DataStream object](https://aind-data-schema.readthedocs.io/en/latest/acquisition.html#datastream). This mapping is critical for correctly interpreting the data, as the `Fiber_n` columns in the CSV files represent patch_cord_n data, and experimenters may not always connect patch_cord_n to fiber_n (e.g., if a particular implanted fiber is damaged or if a different configuration is needed for the experiment). Also documents the intended measurement for each fiber (e.g., norepinephrine, dopamine, calcium) which is stored in the [channel](/allen/aind/stage/vr-foraging/data/) config of each [PatchCordConfig](https://aind-data-schema.readthedocs.io/en/latest/components/configs.html#patchcordconfig), assuming that this information was provided by the scientist in the surgical request.
+
```

Comment:
the link after [channel] looks like a copy/paste mistake

Reply:
Good catch! Fixed in https://github.com/AllenNeuralDynamics/aind-file-standards/pull/59/commits/818070b88477215aaa33cd387464e775ea05f610